openedx/core/djangoapps/team_api/models.py

```diff
@@ -0,0 +1,88 @@
+"""
+Django models related to teams functionality.
+"""
+
+from django.contrib.auth.models import User
+from django.db import models
+from .utils import slugify
+
+from xmodule_django.models import CourseKeyField
+
+
+class CourseTeam(models.Model):
+    """
+    This model represents team related info.
+    """
+
+    class Meta:
+        unique_together = (('team_id', 'course_id'),)
```

Comment:
@andy-armstrong shouldn't team_id be unique across all courses? I think that was the ultimate decision, and @dan-f had a suggestion about creating unique IDs in the comment section of https://openedx.atlassian.net/wiki/display/TNL/Team+API.